packages/debug/src/browser/view/debug-toolbar-widget.tsx

```diff
@@ -77,6 +81,29 @@ export class DebugToolBar extends ReactWidget {
             {this.renderStart()}
         </React.Fragment>;
     }
+
+    protected renderContributedCommands(): React.ReactNode {
+        return this.menuModelRegistry
+            .getMenu(DebugToolBar.MENU)
+            .children.filter(node => node instanceof CompositeMenuNode)
+            .map(node => (node as CompositeMenuNode).children)
+            .reduce((acc, curr) => acc.concat(curr), [])
+            .filter(node => node instanceof ActionMenuNode)
```

Comment:
Use `CommandMenuNode.is` rather than the `instanceof` check, and add the `when` clause check here rather than below in `debugAction`.